db/migrate/20211028084844_add_virtues_to_hero.rb

```diff
@@ -0,0 +1,6 @@
+class AddVirtuesToHero < ActiveRecord::Migration[6.1]
+  def change
+    add_column :heros, :virtue_1_internal_name, :string
```

Comment:
Assuming that we do not want to "snapshot" user choice, then it is so much more useful to store the actual relation between the hero (which is actually a set of user choices) and the virtue itself.

Please define the `has_many :virtues` on the `Hero` model, and make the database columns follow the rails convention here.  A bonus point would be to to define it as the `has_many :virtues, through: :hero_virtues` - making a separate, linking table for these two relations. That would be my aim anyways (please expect a refactoring request like this in the near future :) )